apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/tag/Tags.java

```diff
@@ -83,6 +83,8 @@ private Tags() {
         public static final StringTag PARAMS = new StringTag(11, "http.params", true);
 
         public static final StringTag BODY = new StringTag(13, "http.body");
+
+        public static final StringTag HEADERS = new StringTag(14, "http.headers", true);
```

Comment:
Basically, I don't we don't need `override=true`. Do you have case to write headers more than once?